docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:4.2.5
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8080

```

Comment:
you can change the directory to be in the evergreen folder, which should have manage.py once you move it back there